packages/driver/src/cypress/cy.ts

```diff
@@ -343,13 +346,16 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
     this.ensureNotHiddenByAncestors = ensures.ensureNotHiddenByAncestors
     this.ensureExistence = ensures.ensureExistence
     this.ensureElExistence = ensures.ensureElExistence
+    this.ensureNull = ensures.ensureNull
+    this.ensureNotNull = ensures.ensureNotNull
```

Comment:
same question, do we need to add on the global?

Reply:
Removed these two entirely.